Tests/Api/DataTests.cs

```diff
@@ -0,0 +1,140 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using NUnit.Framework;
+using QuantConnect.Api;
+using QuantConnect.Util;
+
+namespace QuantConnect.Tests.API
+{
+    /// <summary>
+    /// API Data endpoint tests
+    /// </summary>
+    [TestFixture]
+    class DataTests : ApiTestBase
```

Comment:
nit - missing `public`?
There are a few test failing in the CI travis/GH think because of missing ignores that were there before?

Reply:
Yup, need to go back in and place all the ignores 👍🏽